dot/parachain/util/util.go

```diff
@@ -25,3 +30,137 @@ func SigningKeyAndIndex(
 	}
 	return nil, 0
 }
+
```

Comment:
let's create a package called chainapi and put all the chain api related code there.

Reply:
updated.